Spec/Auth.swift

```diff
@@ -188,9 +188,7 @@ class Auth : QuickSpec {
 
                 // RSA4b
                 it("in REST, if the token creation failed or the subsequent request with the new token failed due to a token error, then the request should result in an error") {
-                    let options = AblyTests.clientOptions()
-                    options.key = AblyTests.commonAppSetup().key
-                    options.token = getTestToken()
+                    let options = AblyTests.commonAppSetup()
```

Comment:
This still won't have a means to renew the token or even use token auth, as it will use the key.